sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/BenchmarkRequestSubscriber.java

```diff
@@ -15,26 +18,50 @@
 
 public class BenchmarkRequestSubscriber<T> extends BaseSubscriber<T> {
     final static Logger logger = LoggerFactory.getLogger(BenchmarkRequestSubscriber.class);
+    private final StopWatch durationTimer = new StopWatch();
```

Comment:
StopWatch can be used on different threads and hence it is not thread safe.
Please use a volatile long for measuring the checkpoint.

`private long startTimeInMs`

Reply:
done